tools/vp20compiler/main.c

```diff
@@ -463,6 +466,8 @@ void translate(const char* str)
                 vsh_set_field(vsh_ins, FLD_OUT_ORB, OUTPUT_C);
                 // TODO: the index needs ajustment?
                 vsh_set_field(vsh_ins, FLD_OUT_ADDRESS, ins.DstReg.Index);
+            } else if (ins.DstReg.File == PROGRAM_ADDRESS) {
+                //no need to do anything, setting the MAC_ARL is all that's necessary
```

Comment:
nit: We usually do comments with space after `//` and starting with a capital letter.

Reply:
Fixed it for you